cirq/google/common_serializers.py

```diff
@@ -127,6 +127,54 @@ def _near_mod_2(e, t, atol=1e-8):
 ]
 
 
+#
+# PhasedXZ gate (de)serializer
+# TODO(maffoo): Move to SINGLE_QUBIT_(DE)SERIALIZERS when server support is in.
```

Comment:
Better to file an issue and reference it in place of your username.

Reply:
Done: #2602.